docs/settingUp.md

```diff
@@ -79,6 +100,8 @@ Eclipse IDE is our preferred development environment. Support requests related t
    ./gradlew resetEclipseDeps
    ```
    This command can be run again whenever the dependencies need to be updated.
+  
+ **Verification** : After this the `.classpath` folder is added in the project root directory.
```

Comment:
~~After this~~ The file named `.classpath` ~~folder is~~ should be added ~~in~~ to the project root directory.